app/router.js

```diff
@@ -7,12 +7,25 @@ const SCROLL_RESET_DISABLED_ROUTES = ['preprints.provider.moderation', 'preprint
 const Router = Ember.Router.extend({
     location: config.locationType,
     rootURL: config.rootURL,
+
+    transitioningFrom: '',
+    transitioningTo: '',
+
+    disableResetScroll: Ember.computed('transitioningFrom', 'transitioningTo', function() {
+        return SCROLL_RESET_DISABLED_ROUTES.includes(this.get('transitioningFrom')) &&
+            SCROLL_RESET_DISABLED_ROUTES.includes(this.get('transitioningTo'));
+    }),
+
     willTransition(oldInfos, newInfos) {
         this._super(...arguments);
-        const oldRoute = Ember.Router._routePath(oldInfos);
-        const newRoute = Ember.Router._routePath(newInfos);
-        const disableResetScroll = SCROLL_RESET_DISABLED_ROUTES.includes(oldRoute) && SCROLL_RESET_DISABLED_ROUTES.includes(newRoute);
-        if (!disableResetScroll) {
+
```

Comment:
This is the minorest of minor things, but it bothers me that there's an empty line after `_super` here, but not in `didTransition`.

Reply:
Do you prefer newline or no newline?

Reply:
No real preference besides consistency. I suppose I would probably put a newline?

Reply:
@aaxelb I added one because I like newlines.

Reply:
Glad we're on the same page :1st_place_medal: